mkdocs.yml

```diff
@@ -341,6 +340,9 @@ nav:
               - 'BIOS lock support': unified-test-documentation/dasharo-security/20J-bios-lock-support.md
               - 'SMM BIOS write protection': unified-test-documentation/dasharo-security/20O-SMM-bios-write-protection.md
               - 'Early boot DMA protection': unified-test-documentation/dasharo-security/20L-early-boot-dma-protection.md
+              - 'BIOS Setup password': unified-test-documentation/dasharo-security/20R-bios-setup-password.md
```

Comment:
The nomenclature should be changed in accordance with the `Product IDs` documentation.

Also link to the `Network Boot availability` is broken.

Reply:
Changed